README.EN.md

```diff
@@ -0,0 +1,65 @@
+2. If you want to actively contribute to the design then you can request access to the private Figma by sending a DM to [@jelleprins](https://www.twitter.com/jelleprins) with your email address. Then you can create your own page within Figma.
+3. You can use GitHub to make suggestions or improvements to the contents of this repository.
+
+### Take a look at the latest designs
+
+The latest version of our designs is share in [this Figma](https://www.figma.com/file/EJ4aJwKnemkxysCZ6aAzFv/Covid-19-notificatie-app-(Read-only)). Here you will see the status of each part of the design: which parts are finished and which parts still require some work.
+
+The page with the highest number is the stable version. For each version there is a changelog available where we list the changes that have been made since the previous release.
+
+Comments and answers made in older shared versions will remain available there.
+
+You can contribute by making questions and comments. All of your suggestions are welcome: we'll do our absolute best to read all of you suggestions, answer all of your questions and incorporate all of the good suggestions into the designs. However we cannot promise that all the suggestions will be incorporated.
+
+### Join the discussion on our Slack channel
+
+Questions or reactions via the Slack community of CodeForNL (`#notificatie-app-design` channel). Register via https://doemee.codefor.nl/. Members of the design team follow the discussion and where possible also join in.
+
+#### Contribute to our public challenges
+
+Sometimes even we don't have all of the the answers so we look to you - the public - for help. We convert the design mountains into challenges which you can contribute to in our publically available Figma file. Everyone on their own page. Want to contribute? Then you can request access by sending a Twitter DM to [@jelleprins](https://www.twitter.com/jelleprins).
+
+---
+
+## License (this translation is provisional and still under review)
+
+The intellectual property rights to (part of) this work rest with the Kingdom of the Netherlands. The Kingdom of the Netherlands reserves the right to this (part of the) work.
+
+The Kingdom of the Netherlands may use and publish the designs and documentation to which you contribute. By making a contribution, you transfer the intellectual property and copyright to the Kingdom of the Netherlands and agree that your contribution may also be used and made public by the Kingdom of the Netherlands.
+
+---
+
+NOTE: This README is also available [in English](README.EN.md). Thank you for understanding that the English version might not always be up-to-date with the Dutch one.
+
+NOTE: Deze README is ook beschikbaar [in het Engels](README.EN.md). Dank voor je begrip dat de Engelse versie niet altijd up-to-date is
```

Comment:
In the English Readme, I was expecting a link to the Dutch version. See https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/README.de.md for an example of this.